exercises/sending-signals-external/solution/worker.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+import logging
+
+from activities import PizzaOrderActivities
+from shared import TASK_QUEUE_NAME
+from temporalio.client import Client
+from temporalio.worker import Worker
+from workflow import PizzaOrderWorkflow, FulfillOrderWorkflow
+
+
+async def main():
+    logging.basicConfig(level=logging.INFO)
+    client = await Client.connect("localhost:7233", namespace="default")
+
+    activities = PizzaOrderActivities()
+
+    worker = Worker(
+        client,
+        task_queue=TASK_QUEUE_NAME,
+        workflows=[PizzaOrderWorkflow,FulfillOrderWorkflow],
```

Comment:
```suggestion
        workflows=[PizzaOrderWorkflow, FulfillOrderWorkflow],
```

Would suggest a formatter like black or ruff on all code